workers/tester.sh

```diff
@@ -4,7 +4,7 @@
 
 # This script should always run as if it were being called from
 # the directory it lives in.
-script_directory=`dirname "${BASH_SOURCE[0]}"`
+script_directory=`cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd`
```

Comment:
This is a good point. An alternative that I can think of is:
```
script_directory=`realpath $(dirname ${BASH_SOURCE[0]})`
```
Please feel free to use the one that you like more.